app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+    belongs_to :user
+    belongs_to :post
+
+      validates :text,  presence: true, length: { minimum: 6 }
+      validates :user_id,  presence: true
```

Comment:
The validation for the user_id and post_id are both handled by the associations in lines 2-3 :) Without these two lines 6-7 we still will prevent a comment creation without user_id and post_id

Reply:
Thanks for observations